website/docs/api/example.md

```diff
@@ -286,10 +286,12 @@ Calculate alignment tables between two tokenizations.
 
 ### Alignment attributes {#alignment-attributes"}
 
+Alignment attributes are managed using the `AlignmentArray`, which is a simplified version of Thinc's [Ragged](https://thinc.ai/docs/api-types#ragged) type that only supports the `data` and `length` attributes. 
```

Comment:
Adding a separate container entry for `AlignmentArray` seemed like overkill, so I just added this note instead. Although it closely follows the in-code documentation, it's not 100% accurate as `AlignmentArray` actually contains two additional properties, but none that a user would be likely to want to access. I feel this solution is a good compromise between accuracy and clarity/conciseness.